test/routes/sessions.js

```diff
@@ -178,23 +178,37 @@ describe('API > Sessions', () => {
 
             expect(decodedJWT).to.containSubset({
               id: '125',
-              type: 'users',
-              data: {
-                email: 'user@email.com',
-                first_name: 'user',
-                last_name: 'last',
-                teams: ['Operations'],
-              },
-              relationships: {
-                renderings: {
-                  data: [
-                    {
-                      type: 'renderings',
-                      id: 1,
-                    },
-                  ],
-                },
-              },
+              email: 'user@email.com',
+              first_name: 'user',
+              last_name: 'last',
+              teams: ['Operations'],
+            });
+
+            done();
+          });
+      });
+
+      it('should return a valid jwt (new format)', (done) => {
```

Comment:
[Cleaning] Isn't it a duplicate test? The one above looks very very similar.
So I am pretty sure this one is useless.